app/components/moderation-base/component.js

```diff
@@ -1,30 +1,36 @@
 import { computed } from '@ember/object';
+import { inject as service } from '@ember/service';
 import Component from '@ember/component';
+
 /**
  * The base for the provider moderation page (tabs, provider name, and breadcrumbs).
  *
  * Sample usage:
  * ```handlebars
- * {{moderation-base
- *   provider=provider
- *   pendingCount=pendingCount
- * }}
+ * {{moderation-base}}
  * ```
  * @class moderation-base
  * */
 export default Component.extend({
-    tabs: computed('pendingCount', function() {
+    theme: service(),
+    store: service(),
+
+    tabs: computed('theme.reviewableStatusCounts.pending', function() {
         return [
             {
                 nameKey: 'global.moderation',
                 route: 'preprints.provider.moderation',
                 hasCount: true,
-                count: this.get('pendingCount') || 0,
+                count: this.get('theme.reviewableStatusCounts.pending'),
             },
             {
                 nameKey: 'global.settings',
                 route: 'preprints.provider.settings',
             },
         ];
     }),
+
+    didReceiveAttrs() {
+        this.set('providerName', this.get('theme.provider.name'));
```

Comment:
This would make more sense as a computed alias, since `theme.provider` updating wouldn't rerun `didReceiveAttrs`